CDP4Addin/AddinPluginLoader.cs

```diff
@@ -63,10 +71,19 @@ public AddinPluginLoader(IAppSettingsService<AddinAppSettings> appSettingsServic
         /// <param name="folder">
         /// the folder that contains the CDP4 plugin
         /// </param>
-        private void LoadPlugins(string folder)
+        /// <param name="isNewPlugin">
+        /// the flag telling is plugin is new to the settings
+        /// </param> 
+        private void LoadPlugins(string folder, bool isNewPlugin)
```

Comment:
instead of `string` for folder, pass the `DirectoryInfo` here, so you dont have to `new` it up later. `DirectoryInfo` always has the path in it already.

Reply:
Code has been refactored